native/src/components/SimpleImage.tsx

```diff
@@ -1,46 +1,59 @@
-import React, { ReactElement, useMemo } from 'react'
-import { Image, View, StyleProp, ImageStyle, ImageResizeMode, Platform } from 'react-native'
-import { SvgCssUri } from 'react-native-svg'
+import React, { JSXElementConstructor, ReactElement, useMemo } from 'react'
+import { Image, View, StyleProp, ImageStyle, ImageResizeMode } from 'react-native'
+import { SvgCssUri, SvgProps } from 'react-native-svg'
 import styled from 'styled-components/native'
 
+import { PageResourceCacheStateType } from '../utils/DataContainer'
+import getCachedThumbnail from '../utils/getCachedThumbnail'
+import Icon from './base/Icon'
+
 const StyledImage = styled.Image<{ aspectRatio?: number }>`
   ${props => props.aspectRatio && `aspect-ratio: ${props.aspectRatio};`}
 `
 
-export type ImageSourceType = string | number | null
+type AspectRatioImageProps = {
+  source: string
+  resizeMode: ImageResizeMode
+  specifyAspectRatio: boolean
+  style?: StyleProp<ImageStyle>
+}
+
+const AspectRatioImage = ({ source, style, resizeMode, specifyAspectRatio }: AspectRatioImageProps) => {
+  const aspectRatio = useMemo(() => {
+    let value: undefined | number
+    Image.getSize(source, (width, height) => {
+      value = width / height
+    })
+    return value
+  }, [source])
+
+  return (
+    <StyledImage
+      aspectRatio={specifyAspectRatio ? aspectRatio : undefined}
+      source={{ uri: source }}
+      resizeMode={resizeMode}
+      style={style}
+    />
+  )
+}
+
+export type ImageSourceType = JSXElementConstructor<SvgProps> | string | number | null
 type SimpleImageProps = {
   source: ImageSourceType
   style?: StyleProp<ImageStyle>
   resizeMode?: ImageResizeMode
   // In order to be able to align an image, its width or aspect ratio has to be set
   specifyAspectRatio?: boolean
-}
-
-// For ios you should not use the absolute path, since it can change with a future build version, therefore we use home directory
-// https://github.com/facebook/react-native/commit/23909cd6f62056de0cd0f7c06e3997dd967c139a
```

Comment:
Have you tested with ios, so this still works?

Reply:
I can't don't have an ios device. @LeandraH could you please test this specifically again, i.e. showing images while offline on ios?

Reply:
Checked, it still works. Except for the back icon in the header but that also disappears on main. I'll check that in the production app and open a ticket if necessary